### ISSUE TITLE
keras version ?

### ISSUE THREAD

**Issue by iperov** [michaelrzhang/lookahead#2]
subj

**Comment by michaelrzhang** [comment]
No, we unfortunately do not have one. There are other repos that do.

**Comment by iperov** [comment]
nevermind
learning rate dropout better than lookahead

**Comment by zaccharieramzi** [comment]
There is now an official TensorFlow/Keras implementation if anyone is interested: https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/Lookahead
